src/diffusers/pipelines/pixart_alpha/__init__.py

```diff
@@ -32,7 +33,9 @@
     except OptionalDependencyNotAvailable:
         from ...utils.dummy_torch_and_transformers_objects import *
     else:
-        from .pipeline_pixart_alpha import PixArtAlphaPipeline
+        from .pipeline_pixart_alpha import PixArtAlphaPipeline, ASPECT_RATIO_2048_BIN, ASPECT_RATIO_1024_BIN, \
+            ASPECT_RATIO_512_BIN, ASPECT_RATIO_256_BIN
```

Comment:
fyi, looks like ASPECT_RATIO_2048_BIN is not defined in `pipeline_pixart_alpha` but `pipeline_pixart_sigma`

Reply:
Oh yes! Thanks so much.

Reply:
there seems to be a problem with circular import too, sent you a PR on your fork https://github.com/lawrence-cj/diffusers/pull/12

Reply:
Merged!❤